readme.md

```diff
@@ -137,6 +135,11 @@ Configure it in `package.json`.
 - [regex-shorthand](docs/rules/regex-shorthand.md) - Enforce the use of regex shorthands to improve readability. *(fixable)*
 - [throw-new-error](docs/rules/throw-new-error.md) - Require `new` when throwing an error. *(fixable)*
 
+## Deprecated Rules
+
+|Deprecated rule|Replaced by|Rule Details|
+|:--|:--|:--|
+|[unicorn/prefer-exponentiation-operator](docs/rules/prefer-exponentiation-operator.md)|[prefer-exponentiation-operator](https://eslint.org/docs/rules/prefer-exponentiation-operator)|Prefer the exponentiation operator over `Math.pow()`|
```

Comment:
I don't think we should use a table for this. Markdown tables are annoying to edit and comes with little benefit here. Not every deprecated rule will have a replacement either. Better to just use a plain list.

Reply:
done.